### PR TITLE
footer

### PR DESCRIPTION
**Pull request by oyindamoladekeye**

### CODE REVIEW

File: scrybe_frontend/src/components/Footer/index.jsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ function Footer() {
             </div>
 
             <div className={styles.joinContainer}>
+              <p className={styles.provide}>Join our community:</p>
               <div className={styles.socials}>
```
Comment (dprincecoder) [inline scrybe_frontend/src/components/Footer/index.jsx:34]: Mentor asked us to remove this "Join our community", I removed it, so why did you add it back?

**Review by dprincecoder — CHANGES_REQUESTED** [review]

File: scrybe_frontend/src/components/Footer/index.jsx (hunk 1 above)
Comment (oyindamoladekeye) [reply scrybe_frontend/src/components/Footer/index.jsx:34]: Pls merge and remove it I'm having issues from my side
